src/steps/video-setup/source-select.tsx

```diff
@@ -44,11 +44,20 @@ export const SourceSelection: React.FC<SourceSelectionProps> = ({
 
   const clickBoth = async () => {
     setActiveSource("both");
-    await startUserCapture(dispatch, settings, userConstraints);
-    await Promise.all([
-      queryMediaDevices(dispatch),
-      startDisplayCapture(dispatch, settings, displayConstraints),
-    ]);
+    // Safari needs the display capture to be called first as
+    // otherwise `getDisplayMedia` won't register the button click and
+    // throw an error.
```

Comment:
```suggestion
    // Safari needs the display capture to be called first as otherwise
    // it thinks `getDisplayMedia` is called without user gesture and
    // throws an error.
```

Mentioning "user gesture" here is probably a good idea.